evaluation/TUTORIAL.md

```diff
@@ -31,7 +31,7 @@ workspace_base = "/path/to/your/workspace"
 workspace_mount_path = "/path/to/your/workspace"
 # ==========================
 
-sandbox_container_image = "ghcr.io/opendevin/sandbox:latest"
+sandbox_container_image = "ghcr.io/opendevin/sandbox:main"
```

Comment:
@li-boxuan here?

Reply:
oh yes we should remove this too, thanks!